tutorials/README.md

```diff
@@ -0,0 +1,26 @@
+# Step by step tutorials
+
+## Setup
+
+For running the examples in the tutorials you can either manually install Sockeye, it's dependencies and additional
+tools used or use the Dockerfile we provide, which comes with all batteries included.
+
+### Either: Dockerfile
+
+Coming soon...
+
+<!--we provide a docker file that already includes all prerequisites for running the tutorials.
+If you're using the provided Dockerfile simply skip the `Setup` sections in the tutorials below.-->
+
+### Or: Manual installation
+If you do not use the provided Dockerfile you can follow the [installation instructions](../README.md#installation)
+for how to manually install Sockeye and all dependencies. The tutorials below might have additional dependencies that
+will mentioned at the beginning of each tutorial.
+
+## Tutorials
+
+Below is the full list of tutorials we provide. We recommend going through them in order as they will gradually
+introduce the different concepts and parameters used for training and translation.
```

Comment:
"introduce different concepts and parameters [...]"

Reply:
changed